marge/bot.py

```diff
@@ -104,12 +106,14 @@ def _run(self, repo_manager):
                         options=self._config.merge_opts,
                     )
                     merge_job.execute()
-                    time_to_sleep_in_secs = 5
                 else:
                     log.info('Nothing to merge at this point...')
-                    time_to_sleep_in_secs = 30
                 log.info('Sleeping for %s seconds...', time_to_sleep_in_secs)
-                time.sleep(time_to_sleep_in_secs)
+                time.sleep(time_to_sleep_between_projects_in_secs)
+            time.sleep(max(0,
+                           min_time_to_sleep_after_iterating_all_projects -
+                           time_to_sleep_between_projects_in_secs * len(filtered_projects)))
```

Comment:
@aschmolck As it is, we are going to let the user know that marge is about to sleep for 1 sec, and fail to log anything when sleeping for 30 secs. This should be the other way round.

Reply:
Done